app/(gcforms)/[locale]/(user authentication)/auth/register/action.ts

```diff
@@ -58,6 +58,7 @@ const validate = async (
         ),
       ]),
       password: v.string([
+        v.minLength(1, t("input-validation.required", { ns: "common" })),
```

Comment:
Do we still need this one, even though we have another minimum length check below?

Reply:
Yes, they drive different validation error messages.  The min 1 drives the 'field required' and the min 8 drives 'password must have at least 8 characters'